Write numbers as numbers

protobi opened the pull request:
Updated mocha `test/example.js` as well.

protobi-pieter committed:
Write numbers as numbers

protobi-pieter committed:
Write numbers as numbers

mattwilsn commented:
https://github.com/chuanyi/msexcel-builder/issues/49

I opened this as an issue. 

Looking forward to this update

protobi-pieter committed:
Merge remote-tracking branch 'upstream/master'

protobi-pieter committed:
Update test

protobi-pieter committed:
Update test

protobi-pieter committed:
Apply rgb fill and font colors

protobi-pieter committed:
Apply rgb fill and font colors

protobi-pieter committed:
Add autofilter; write zero values to cells

protobi-pieter committed:
Add autofilter; write zero values to cells

protobi-pieter committed:
Add colors, autofilter

protobi-pieter committed:
Add colors, autofilter

protobi-pieter committed:
Update tests to write output files to ./test/out

protobi-pieter committed:
Update tests to write output files to ./test/out

protobi-pieter committed:
Clean sheet names of disallowed characters

protobi-pieter committed:
Clean sheet names of disallowed characters

protobi-pieter committed:
generate named range only if autoflter applied

protobi-pieter committed:
generate named range only if autoflter applied

protobi-pieter committed:
generate named range only if autoflter applied

protobi-pieter committed:
generate named range only if autoflter applied

protobi-pieter committed:
Update tests to reflect named ranges; bump package version

protobi-pieter committed:
Update tests to reflect named ranges; bump package version

dcrystalj commented:
@protobi 
Do you know how to add formula format as well like `=(A1+ A2)`?

mattwilsn commented:
@dcrystalj you need to to do a manual calculations 

`sheet.set(col, row, 1 + 4)`

I do not think there is a way to do any calculations on cells that are already filled

dcrystalj commented:
@mattwilsn I need to have formula so others can make some easy changes in excel later. I saw it is possible in [exceljs](https://github.com/guyonroche/exceljs) with value type `Excel.ValueType.Formula`. For me it is quite hard to change whole lib. That is the reason why I am asking for help here

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
Add rudimentary number format

protobi-pieter committed:
Add rudimentary number format

protobi-pieter committed:
Add rudimentary number format

protobi-pieter committed:
Add rudimentary number format

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
Port to JSZip 3.1.2 with async methods

protobi-pieter committed:
Port to JSZip 3.1.2 with async methods

protobi-pieter committed:
add concise setter

protobi-pieter committed:
add concise setter

protobi-pieter committed:
add concise setter

protobi-pieter committed:
add concise setter

protobi-pieter committed:
Take Date values directly

protobi-pieter committed:
Take Date values directly

protobi-pieter committed:
Update style test

protobi-pieter committed:
Update style test

protobi-pieter committed:
Add date formats, composite set

protobi-pieter committed:
Add date formats, composite set

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
Autofilter more than 26 columns

kokeksibir committed:
Updated section describing JSZip generate

JSZip api is changed, now `generateAsync` should be used instead of `generate`.

protobi-pieter committed:
Merge pull request #2 from kokeksibir/patch-1

Updated section describing JSZip generate

protobi-pieter committed:
Take filename on save, rather than in constructor

protobi-pieter committed:
Take filename on save, rather than in constructor

protobi-pieter committed:
Merge branch 'master' of https://github.com/protobi/msexcel-builder

protobi-pieter committed:
Update README.md

Kerry-G commented:
@dcrystalj I know it's been a while, but did you ever found a solution to write function with this lib?

dcrystalj commented:
@Kerry-G i changed library :-1:

protobi-pieter committed:
Package for use in browser

protobi-pieter committed:
Update dependencies

Commit:
Exclude fs and path dependencies in browser

protobi-pieter committed:
Merge pull request #8 from ArthurBrito/7-Compability-Browser-ng6

Exclude fs and path dependencies in browser

Commit:
Update repository url

Commit:
Fixed #14
Added custom number formatters
Fixed git URL
Bumped version

Commit:
Fixed #14
Added custom number formatters
Fixed git URL
Bumped version

Commit:
bump version

Commit:
Attempting to fix number formats

Commit:
Fixing invalid number format id

Commit:
Added custom number formats when converting to xml

Commit:
Added custom number formats when converting to xml

Commit:
Version bump

Commit:
Set defaults to english
Bumped version

protobi-pieter committed:
Merge pull request #15 from JonnoFTW/master

Fixed number formats

valery-vitko committed:
Support xmlbuilder >=11 (fixes #16)

- fixes error about non-trimmed identifiers ('Invalid character in name: fileVersion ').

protobi-pieter committed:
add ability to remove gridlines, set other properties

protobi-pieter committed:
Provide way to turn off gridlines, etc. per @saravmajestic https://github.com/chuanyi/msexcel-builder/issues/59

protobi-pieter committed:
Add Sheet.split(ncols,nrows)

protobi-pieter committed:
Add cell formulas

Add cell formulas

Add cell formulas

protobi-pieter committed:
Verify tests

protobi-pieter committed:
Update https://github.com/protobi/msexcel-builder/pull/17

protobi-pieter committed:
Save compressed

protobi-pieter committed:
Add border color

protobi-pieter committed:
Add page breaks; repeating rows

protobi-pieter committed:
Merge pull request #9 from ArthurBrito/repository-info

Update repository url

protobi-pieter committed:
Add print settings

protobi-pieter committed:
Merge branch 'master' of github.com:protobi/msexcel-builder

protobi-pieter committed:
Add print settings

Fa-b committed:
update xmlbuilder dep
once again remove whitespaces from fileVersion and workbookView

Fa-b committed:
add font styles:
underline
strike
outline
shadow

Fa-b committed:
fix left border

Fa-b committed:
Update README.md

protobi-pieter committed:
Merge pull request #21 from Fa-b/master

fixes & style additions

protobi-pieter committed:
Update page margin

protobi-pieter committed:
Merge branch 'master' of github.com:protobi/msexcel-builder

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
Add image support

protobi-pieter committed:
Introduce addImage

protobi-pieter committed:
Extend .set to take an array of array or nested object

protobi-pieter committed:
Update per https://github.com/chuanyi/msexcel-builder/issues/43

protobi-pieter committed:
Update to add notes

protobi-pieter committed:
Update notes to add multiple lines, formats

protobi-pieter committed:
Extend page breaks to max column/row

dcrystalj commented:
why not merging this PR after so many commits ?

protobi-pieter committed:
Add Workbook.set() method to allow workbook to be generated from JSON data

protobi-pieter committed:
Add Workbook.set() method to allow workbook to be generated from JSON data

protobi-pieter committed:
Update create from pojo

protobi-pieter committed:
Bug fix

protobi-pieter committed:
Extend set by value

protobi-pieter committed:
Update set from data objects

protobi-pieter committed:
Update set from data objects

protobi-pieter committed:
Update tests

protobi-pieter committed:
Update tests

protobi-pieter committed:
#23 remove control characters from strings

protobi-pieter committed:
Remove prototype pollution

protobi-pieter committed:
Update #4

protobi-pieter committed:
Add Worksheet.text(col, row, str) alias

protobi-pieter committed:
Add Worksheet.text(col, row, str) alias

protobi-pieter committed:
Update package-lock; test/notes

protobi-pieter committed:
Update package-lock; test/notes

protobi-pieter committed:
Test >25 cols, >1000 rows

protobi-pieter committed:
Update README

protobi-pieter committed:
Warn rather than throw exception on invalid number format

protobi-pieter committed:
Warn rather than throw exception on invalid number format

protobi-pieter committed:
Fix image support

protobi-pieter committed:
Fix image support
